src/test/java/seedu/address/logic/commands/AddVisitCommandTest.java

```diff
@@ -21,7 +27,7 @@
 public class AddVisitCommandTest {
     @Test
     public void constructor_nullLocation_throwsNullPointerException() {
```

Comment:
Typo here. Shouldn't be nullLocation but instead the parameter that is null in this method.

Reply:
Have changed the method name